tests/platform_tests/conftest.py

```diff
@@ -156,11 +156,12 @@ def get_report_summary(duthost, analyze_result, reboot_type, reboot_oper, base_o
         if lacp_sessions_dict and "lacp_sessions" in lacp_sessions_dict else None
     controlplane_summary = {"downtime": "",
                             "arp_ping": "", "lacp_session_max_wait": ""}
-    if lacp_sessions_waittime and len(lacp_sessions_waittime) > 0:
-        max_lacp_session_wait = max(list(lacp_sessions_waittime.values()))
-        analyze_result.get(
-            "controlplane", controlplane_summary).update(
-                {"lacp_session_max_wait": max_lacp_session_wait})
+    if duthost.facts['platform'] != 'x86_64-kvm_x86_64-r0':
```

Comment:
Check for LACPDU timing also happen at advanced-reboot test. Shouldn't we disable the check there too?
https://github.com/sonic-net/sonic-mgmt/blob/1069b2257a712281f77f57eeb34a46e78cc4eb66/ansible/roles/test/files/ptftests/advanced-reboot.py#L1631

Also w/ teamd retry count changes, these checks should still be done for KVM DUT when peer is SONiC T1. @saiarcot895  can you confirm?

Reply:
I would think that even without the teamd retry count changes or with cEOS neighbors, LACP session timings can be captured. Dataplane would be busted anyways, since this is KVM, but it would validate that control plane isn't down for too long.

The 90-second check would probably need to be bumped up in the case of SONiC neighbors with teamd retry count feature, but I don't have a completely reliable check for that at the moment.

Reply:
Correction: actually checking for this on KVM would result in too many false positives, since the LACP downtime without my changes is around 90 seconds, so it should still be disabled in the general case.

For SONiC neighbors and teamd retry count feature present, it can probably be enabled for that, but there would need to be a feature check for that somehow.